openapi.yaml

```diff
@@ -531,7 +531,7 @@ paths:
 
 components:
   schemas:
-    RootMessage:
+    AggregatorFeaturesMessage:
       description: Aggregator public information
```

Comment:
Can you add a bit more information in the description?

Reply:
Done ✅